### PR TITLE
change from 3 digits to 6 digits in print out

### PR DESCRIPTION
**Pull request by mjburton11**
Increase the number of digits in the gpfit print out from 3 to 6.  Increases fit accuracy.

### CODE REVIEW

**Comment by mjburton11** [comment]
@bqpd I need help making this nice and better before merging.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+"""Standard Python setup script for GPkit Models"""
```
Comment (bqpd) [inline setup.py]: GPkit Models -> GPfit?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+from __future__ import print_function
+import sys
+from distutils.core import setup
+
+LONG_DESCRIPTION = """
+GPkit Models is a library of geometric programming and signomial programming
+models that can be manipulated and solved using
+`GPfit <https://github.com/hoburg/gpfit/>`_.
```
Comment (bqpd) [inline setup.py]: lol waht

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+SOFTWARE."""
+
+setup(
+    name="gpfit",
+    description="program to convert data into GP compatible functions that "
+                "can be used with GPkit.",
+    author="MIT Department of Aeronautics and Astronautics",
+    author_email="gpfit@mit.edu",
+    url="https://www.gifhub.com/hoburg/gpfit",
+    install_requires=["numpy", "scipy", "pint"],
```
Comment (bqpd) [inline setup.py]: does it need scipy and pint?

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE."""
+
+setup(
+    name="gpfit",
+    description="program to convert data into GP compatible functions that "
+                "can be used with GPkit.",
+    author="MIT Department of Aeronautics and Astronautics",
+    author_email="gpfit@mit.edu",
```
Comment (bqpd) [inline setup.py]: do we have this email address?

Comment (mjburton11) [inline setup.py]: We don't, so I deleted it.

File: setup.py (hunk 3 above)
Comment (mjburton11) [inline setup.py]: So I saw scipy and left it in.  Deleted pint.

**Comment by whoburg** [comment]
I'm going to try to rebase this to fix the merge conflicts.

**Comment by whoburg** [comment]
@mjburton11, gpfit alreadsy has a `setup.py`, so I'm dropping the two commits related to that

**Comment by whoburg** [comment]
just force pushed, please delete and re-fetch any local copies of the `sigfigs` branch

**Comment by whoburg** [comment]
test this please

**Comment by galbramc** [comment]
test this please

**Comment by galbramc** [comment]
I realized I needed to update the gpfit scripts for mosek 8. I think I just got it.

**Comment by galbramc** [comment]
test this please

**Comment by whoburg** [comment]
pylint is failing with 0 errors, 0 warnings?

**Comment by galbramc** [comment]
test this please

**Comment by galbramc** [comment]
hmm, it's not triggering the build for some reason...

**Comment by whoburg** [comment]
just checked and acdl-jenkins does have admin access

**Comment by galbramc** [comment]
can you try pushing something to this PR

**Comment by galbramc** [comment]
very strange... let me reboot jenkins

**Comment by galbramc** [comment]
test this please

**Comment by galbramc** [comment]
not sure why this isn't starting, but Jenkins might thing this PR is closed for some reason... That's all I can see from the log.

**Comment by whoburg** [comment]
hm, should I try cherry-picking these commits to a new PR?

**Comment by galbramc** [comment]
Worth a try

**Comment by galbramc** [comment]
Maybe closing and reopening the PR?

**Comment by whoburg** [comment]
add to whitelist

**Comment by galbramc** [comment]
That seems to have fixed it

**Comment by galbramc** [comment]
Test this please

**Comment by whoburg** [comment]
easy there jenkins

**Comment by whoburg** [comment]
@galbramc, why the disagreement in pylint status between macys and Reynolds?

**Comment by galbramc** [comment]
That's what that did. There was a new feature in the plugin to apparently show every single build in the matrix. That might be too much. Want me to turn it off?

**Comment by galbramc** [comment]
I'm going to turn that off again. Looks like information overload.

Looks like the counts are off for Reynolds. Could you create an "CI test" PR so I can reset the counts.

**Comment by whoburg** [comment]
#58

**Comment by galbramc** [comment]
Test this please

**Comment by galbramc** [comment]
Alright, looks like you are good to go.

**Comment by whoburg** [comment]
yes -- thanks!

**Comment by bqpd** [comment]
@mjburton11, it's all you.

**Comment by whoburg** [comment]
@mjburton11, please "squash and merge" as opposed to creating a merge commit (if ready to merge)

**Comment by mjburton11** [comment]
I am ready to merge.  How do I "squash and merge"?

**Comment by bqpd** [comment]
Click the arrow next to 
screenshot_2016-12-03_23-21-19
and click on "Squash and Merge", which will turn all this into a single commit. Edit that commit message to your liking...

**Comment by bqpd** [comment]
@mjburton11, the last step is to delete the branch.

**Comment by mjburton11** [comment]
right... sorry

**Pull request merged**